lib/spack/spack/environment/environment.py

```diff
@@ -63,7 +63,10 @@
 
 
 #: path where environments are stored in the spack tree
-env_path = os.path.join(spack.paths.var_path, "environments")
+# circular dependency to use canonicalize_path but we want to ignore an active env anyways
+# env_path = spack.config.get(os.path.expandvars(os.path.expanduser("config:environments_root")))
+def env_path():
```

Comment:
Does anyone know of a way to do a module level property?  That seems like it would be a better replacement than this.